ktor-server/ktor-server-plugins/ktor-server-thymeleaf/jvm/test/io/ktor/server/thymeleaf/ThymeleafTest.kt

```diff
@@ -17,34 +19,36 @@ import java.util.*
 import java.util.zip.*
 import kotlin.test.*
 
-@Suppress("DEPRECATION")
 class ThymeleafTest {
     @Test
     fun testName() {
-        withTestApplication {
-            application.setUpThymeleafStringTemplate()
-            application.install(ConditionalHeaders)
-            application.routing {
-                val model = mapOf("id" to 1, "title" to "Hello, World!")
+        testApplication {
```

Comment:
please use `fun testName() = testApplication {` to avoid extra tabulation

Reply:
Thanks for the review, I've removed extra tabulation